site/es/blog/new-in-devtools-92/index.md

```diff
@@ -0,0 +1,224 @@
+---
+layout: "layouts/blog-post.njk"
+title: "Qué hay de nuevo en las DevTools (Chrome 92)"
```

Comment:
Because DevTools is a proper noun, I suggest to omit the definite article 'las': 
```suggestion
title: "Qué hay de nuevo en DevTools (Chrome 92)"
```